### ISSUE TITLE
Amplify hosting build error out of nowhere

### ISSUE THREAD

**Issue by jimenaCabrejas** [aws-amplify/amplify-hosting#3095]
### Before opening, please confirm:

- [X] I have checked to see if my question is addressed in the [FAQ](https://github.com/aws-amplify/amplify-hosting/blob/master/FAQ.md).
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-hosting/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-hosting/blob/master/CONTRIBUTING.md).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
- [X] I have removed any sensitive information from my code snippets and submission.

### App Id

d2vj4wneivrnf

### AWS Region

eu-west-1

### Amplify Hosting feature

Frontend builds

### Describe the bug

I have been working on my app for a couple of months now and my backend is fully deployed and working. Today I made a visual change on my React application and pushed it to Git so the Amplify hosting would update the Frontend of the app. 

The react app is compiling and working correctly locally. But the Amplify hosting console cannot build the app. Here are the error logs:
[log.txt](https://github.com/aws-amplify/amplify-hosting/files/9971867/log.txt)

More over, I think the Amplify Hosting also updated my backend resources. Until now, to update my backend resources (API and Lambda function) I used the CLI, but this time, my Lambda has been updated without me using the CLI. Is this possible?

### Expected behavior

Amplify hosting builds the app correctly

### Reproduction steps

Push the code to git
Wait for Amplify Hosting to fail on the build

### Log output

<details>

```
# Put your logs below this line


```

</details>

**Comment by natoszme** [comment]
Hi there! Something similar happening here. Could you go to #3096 and check whether you have the same differences between the successful and unsuccessful builds? I think it'd help.

**Comment by hloriana** [comment]
Hi @jimenaCabrejas 👋🏽 According to this error in your build logs it looks like you need to update your dependencies with `npm install` and push a new commit:

```
`npm ci` can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
```

**Comment by jimenaCabrejas** [comment]
Hi! I did as you said I executed npm install in my application. But when trying to upload it to git it says that Everything is up-to-date, so I'm guessing my dependencies where already updated.

**Comment by hloriana** [comment]
@jimenaCabrejas Thanks for the update. We also recently updated our build images to use Node.js 16 as the default which seems like that is when your builds starting failing as well. You can downgrade the Node.js version by adding a package version override via the console. Try setting Node.js to 14 or whatever version you're using locally: [Live package updates](https://docs.aws.amazon.com/amplify/latest/userguide/custom-build-image.html#setup-live-updates)

<img width="397" alt="Screen Shot 2022-11-09 at 11 28 37 AM" src="https://user-images.githubusercontent.com/76065904/200900256-f13da0ab-e47b-494c-a088-1bde81a0e5c0.png">

**Comment by riguy14** [comment]
We also just started getting errors in the past day. Ours was due to node-sass needing a specific version of Node installed (e.g. 14, not 16). Overriding the Node version to 14 worked as described above worked.

@hloriana  - Is there a way to know when AWS is changing something so core to the default package like that? Mailing list or something? Swapping out Node versions can cause huge issues with CI/CD so I'm surprised it just gets changed without any warning.
